apps/schedules/lib/repo.ex

```diff
@@ -11,7 +11,7 @@ defmodule Schedules.Repo do
 
   @default_timeout 10_000
   @default_params [
-    include: "trip",
+    include: "trip,trip.occupancies",
```

Comment:
Nice find! Does this fail gracefully if the experimental header is **not** enabled?